docs/api-v1.md

```diff
@@ -549,6 +549,7 @@ Base endpoint is: `/ocs/v2.php/apps/spreed/api/v1`
 
 * `joined_call` - {actor} joined the call
 * `left_call` - {actor} left the call
+* `call_ended` - Call with {user1}, {user2}, {user3}, {user4} and {user5} (Duration 30:23)
```

Comment:
Shouldn't it be `Duration {duration}` instead?

Reply:
No, similar to the room name on rename its not a placeholder (not a known rich object string type)